packages/tomls/src/core/configs/tokens/rusd.toml

```diff
@@ -8,6 +8,6 @@ args = [
 ]
 depends = [
     'invoke.upgrade_core_proxy',
-    'invoke.upgrade_rUSD_proxy',
+    'var.rusd_complete',
```

Comment:
what's this?

Reply:
because initialization of rUSD was skipped for mainnet, we need to do this for testnet. However, the other invokes depend on upgrade proxy rather than initialization, which is not correct for testnet. This var.rusd_complete flags that upgrade is done for mainnet and initialization is done for testnet